src/components/dashboard/Relaying/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { WidgetBody, WidgetContainer } from '@/components/dashboard/styled'
+import { Box, Card, Divider, Grid, Link, Skeleton, SvgIcon, Typography } from '@mui/material'
+import RelayerIcon from '@/public/images/common/relayer.svg'
+import css from './styles.module.css'
+import useRemainingRelays from '@/hooks/useRemainingRelays'
+import useChains, { useCurrentChain } from '@/hooks/useChains'
+import { FEATURES, hasFeature } from '@/utils/chains'
+import ImageFallback from '@/components/common/ImageFallback'
+
+const FallbackChainIcon = ({ color }: { color: string }) => (
+  <div
+    style={{
+      width: '16px',
+      height: '16px',
+      backgroundColor: color,
+      borderRadius: '50%',
+    }}
+  />
+)
+
+const Relaying = () => {
+  const [remainingRelays] = useRemainingRelays()
```

Comment:
This makes two requests on page load, one of which is incorrect: 
![image](https://user-images.githubusercontent.com/20442784/226342787-c10cff1b-daba-4078-9045-e29b564b28c7.png)